ui/pages/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -83,8 +83,8 @@ import {
   TransactionStatus,
   TransactionType,
 } from '../../../shared/constants/transaction';
-import { getTokenAddressParam } from '../../helpers/utils/token-util';
 import { calcGasTotal } from '../../../shared/lib/transactions-controller-utils';
+import { getTokenAddressParam } from '../../helpers/utils/token-util';
```

Comment:
Was this just a change in import order, if so was it intentional? if not i'd recommend trimming out this change to keep history a bit cleaner.

Reply:
That was not intentional! changing that 🙏